src/components/containers/EventCard/EventCard.module.scss

```diff
@@ -5,15 +5,44 @@
 .card {
     --padding: 14px;
 
+    background-position: left;
+    background-size: 51%, 50%;
+    background-repeat: no-repeat;
+
     box-sizing: border-box;
-    position: relative;
 
     margin: 8px 0;
     min-height: 113px;
-    padding: var(--padding) var(--padding) 48px;
+    padding: 14px;
```

Comment:
use variable here

Reply:
i removed this variable, 
after rework it was used in only one place